src/platforms/apple/common/install/swift-package-manager.mdx

```diff
@@ -3,12 +3,6 @@ title: Swift Package Manager (SPM)
 description: "Integrate Sentry into your Xcode project using Swift Package Manager (SPM)."
 ---
 
-To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Swift Packages` -> `Add Package Dependency`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or branch) on the next page.
+To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` on the top-right search field. Define you dependecy rule by selecting the SDK version (or branch), then press "Add Package" button.
 
-When using SPM version 5.2 or later, you must specify the the target name too:
```

Comment:
Isn't this important anymore?

Reply:
There is no way of doing this anymore with Xcode

Reply:
Is it still necessary when using older Xcode versions? if so, we should add a note that this is only for older Xcode versions and not remove it.

Reply:
Good point. And I believe is possible to open a older project in the new xCode.